### PR TITLE
Added 'Choose file' button + Success message

### PR DESCRIPTION
**Pull request by mpbasto**
Hi!
I've added a button where the user can get the file path by browsing through their machine and selecting a file. Also, I added a message after conversion to inform the user that the output was saved into a *.txt file.

Let me know what you think.

Thanks!

### CODE REVIEW

File: gui.py [hunk 1]
```diff
@@ -17,25 +20,44 @@ def main():
...
+    label1.place(x=140, y=100)
 
-    entry1 = Entry(root, bd=5, font=(15), width=70)
+    entry1 = Entry(root, bd=5, font=(15), width=60)
     entry1.bind(main)
-    entry1.place(x=80, y=100)
+    entry1.place(x=140, y=140)
+
+    browse_btn = tk.Button(root, text='Choose File', font=("Arial", 16), pady=6,
+                           width=7, bd=-2, command=browsePic)  # Button to browse through directories
```
Comment (deepaksaipendyala) [inline gui.py:52]: - Choose file text going out of box, keep width 10

File: gui.py [hunk 2]
```diff
@@ -17,25 +20,44 @@ def main():
...
+
+    convrt_btn = Button(root, text="Convert", font=("Arial", 20),
+                        width=10, padx=5, pady=5, command=main)
+    convrt_btn.place(x=340, y=180)
 
-    button1 = Button(root, text="Convert", font=("Times", 20), width=10, padx=5, pady=5, command=main)
-    button1.place(x=310, y=150)
+    success = Label(root, text='', bg='black')
+    success.pack(pady=5)
+    success.place(x=260, y=220)
```
Comment (deepaksaipendyala) [inline gui.py:62]: - Try on working for saving output.txt as optional ' Add a button for it'

Comment (mpbasto) [reply gui.py:62]: Isn't that what the 'Convert' button does already?

Ideally one would change the name of the 'Convert' button to 'Save as txt file' and create a way to display the ascii conversion in the GUI itself. Or is that not possible?

Thanks

**Review by mpbasto — COMMENTED** [review]
Will change the width of the button to 10 and explore the last suggestion further 🥸

File: gui.py (hunk 2 above)
Comment (deepaksaipendyala) [reply gui.py:62]: - Convert button converts and prints ascii art as stdout
- According to #12, Output file should be optional, not default

Comment (deepaksaipendyala) [reply gui.py:62]: - For the purpose of displaying the ascii art in gui, you can save the file. NP

Comment (deepaksaipendyala) [reply gui.py:62]: - stop saving output as default for now. I'll merge this pr, you can work on gui display and submit another pr

Comment (mpbasto) [reply gui.py:62]: Thanks! Already working on it. In the meantime, I update the width.

**Review by deepaksaipendyala — APPROVED** [review]

**Pull request merged**